repair/row_level.cc

```diff
@@ -1113,7 +1127,7 @@ class repair_meta {
         });
     }
 
-    void handle_mutation_fragment(mutation_fragment& mf, size_t& cur_size, size_t& new_rows_size, std::list<repair_row>& cur_rows) {
+    void handle_mutation_fragment(mutation_fragment& mf, size_t& memory_size, size_t& new_rows_size, std::list<repair_row>& cur_rows) {
```

Comment:
The name memory_size is not better than cur_size which stands for the current size of the row buffer.